code/game/gamemodes/scoreboard.dm

```diff
@@ -33,76 +59,70 @@
 	for(var/i in total_antagonists)
 		log_game("[i]s[total_antagonists[i]].")
 
-	// Score Calculation and Display
+	//Score Calculation and Display
 
-	// Who is alive/dead, who escaped
-	for (var/mob/living/silicon/ai/I in mob_list)
-		if (I.stat == 2 && I.z == 1)
-			score["deadaipenalty"] = 1
-			score["deadcrew"] += 1
+	//Run through humans for diseases, also the Clown
+	for(var/mob/living/carbon/human/I in mob_list)
+		if(I.viruses) //Do this guy have any viruses ?
+			for(var/datum/disease/D in I.viruses) //Alright, start looping through those viruses
+				score["disease"]++ //One point for every disease
 
-	for (var/mob/living/carbon/human/I in mob_list)
-//		for (var/datum/ailment/disease/V in I.ailments)
-//			if (!V.vaccine && !V.spread != "Remissive") score["disease"]++
-		if (I.stat == 2 && I.z == 1) score["deadcrew"] += 1
-		if (I.job == "Clown")
+		if(I.job == "Clown")
 			for(var/thing in I.attack_log)
-				if(findtext(thing, "<font color='orange'>")) score["clownabuse"]++
-
+				if(findtext(thing, "<font color='orange'>")) //I just dropped 10 IQ points from seeing this
+					score["clownabuse"]++
 
 	for(var/mob/living/player in mob_list)
-		if (player.client)
-			if (player.stat != 2)
-				var/turf/location = get_turf(player.loc)
-				var/area/escape_zone = locate(/area/shuttle/escape/centcom)
-				if (location in escape_zone)
+		if(player.client)
+			if(player.stat != DEAD)
+				var/turf/T = get_turf(player)
+				if(istype(T.loc, /area/shuttle/escape/centcom) || istype(T.loc, /area/shuttle/escape_pod1/centcom) || istype(T.loc, /area/shuttle/escape_pod2/centcom) || istype(T.loc, /area/shuttle/escape_pod3/centcom) || istype(T.loc, /area/shuttle/escape_pod5/centcom))
```

Comment:
Does that mean cult code is good.

Sorry but copying from caelcode isn't an excuse to write cael tier code.